### PR TITLE
Add converted files

### PR DESCRIPTION
**Pull request by karst**
I have converted all these and edited the object.json files to the standard of the other already included files.

I have converted all the files that were listed on the wiki for things that should be featured in the game.

### CODE REVIEW

**Comment by karst** [comment]
Do I need to update the theme tabs aswell to include these objects? (FE Pirate Style)

**Comment by SpartanFrederic104** [comment]
Yes, you will need to add the "sceneryGroup" field to the .json files (rct2.scenery_group.scgpirat.json, rct2.scenery_group.scgabstr.json, rct2.scenery_group.scgmine.json, rct2.scenery_group_scgjungl.json, etc.)

Also, toontowner.scenery_wall.xxwlbr04 should not be named "Wall", but "Brick Wall with Portcullis Door" and toontowner.scenery_wall.xxwlbr03 should be "Brick Wall with Door".

File: objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr04/object.json [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+            "srcY": 0,
+            "srcWidth": 22,
+            "srcHeight": 39,
+            "x": 10,
+            "y": -23
+        }
+    ],
+    "strings": {
+        "name": {
+            "en-GB": "Wall",
```
Comment (SpartanFrederic104) [inline objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr04/object.json:346]: Brick Wall with Portcullis Door

File: objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr03/object.json [hunk 2]
```diff
@@ -0,0 +1,364 @@
...
+            "srcY": 0,
+            "srcWidth": 20,
+            "srcHeight": 40,
+            "x": 12,
+            "y": -24
+        }
+    ],
+    "strings": {
+        "name": {
+            "en-GB": "Wall",
```
Comment (SpartanFrederic104) [inline objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr03/object.json:346]: Brick Wall with Door

File: objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr04/object.json (hunk 1 above)
Comment (karst) [reply objects/official/scenery_wall/toontowner.scenery_wall.xxwlbr04/object.json:346]: Do you have the translations for other languages for me aswell?

**Comment by Gymnasiast** [comment]
@karst I need to take a good look at that list some time. I might not want to make all the objects into official ones, or put some in an optional pack.

Nevertheless, I think we can proceed with the pirate roofs, with the exception of the one with the chimney. Could you create a separate PR for those? (And leave this one open?)

**Comment by karst** [comment]
Absolutely, will do!

**Comment by Gymnasiast** [comment]
Great, I think we can also proceed with the glass roofs.

**Comment by karst** [comment]
Alright, i'll make a PR right away

**Comment by Gymnasiast** [comment]
Good, let’s also proceed with acww33 and acwwf32.

**Comment by Gymnasiast** [comment]
I think we can proceed with the wooden roofs, in a PR of their own (like we did with the all the others).

**Comment by Gymnasiast** [comment]
OK, I think this is enough for now. We can always revisit individual objects if there is demand.

**Pull request closed without merging**